app/substrait_consumer_function_compatibility.py

```diff
@@ -10,7 +10,9 @@
 
 def support_matrix_df():
     return (
-        ibis.read_csv("./consumer_results.csv")
+        ibis.read_csv(
+            "https://raw.githubusercontent.com/substrait-io/consumer-testing/main/app/consumer_results.csv"
```

Comment:
Does this need to be an externally accessible URL or does it merely need to be in URI form?  I'm surprised that the website does calculations -- I would have expected it to work off precomputed results.

Reply:
I think it needs to be an externally accessible URL.  Maybe at least for their free community cloud.  I think they just access this script and try to run it from their cloud.  I'll get this in and try some other things.